promotion/suggestions.go

```diff
@@ -30,8 +30,9 @@ type CredentialBinding struct {
 
 // Suggestion encapsulates information from the user about where /how they want to contribute
 type Suggestion struct {
-	Type    string `json:"type" valid:"in(auto-contribute|oneoff-tip|recurring-tip)"`
-	Channel string `json:"channel" valid:"-"`
+	Type    string     `json:"type" valid:"in(auto-contribute|oneoff-tip|recurring-tip|payment)"`
```

Comment:
Are these values enumerated as Suggestion Types in the codebase?  I think we use them as strings everywhere...

Reply:
Nope! But it seems like the "auto-contribute" is just used in test values. I'm not too concerned with making these constants yet